Lib/test/test_ssl.py

```diff
@@ -1822,9 +1822,13 @@ def test_connect(self):
         with test_wrap_socket(socket.socket(socket.AF_INET),
                             cert_reqs=ssl.CERT_REQUIRED,
                             ca_certs=SIGNING_CA) as s:
-            s.connect(self.server_addr)
-            self.assertTrue(s.getpeercert())
-            self.assertFalse(s.server_side)
+            try:
+                s.connect(self.server_addr)
+                self.assertTrue(s.getpeercert())
+                self.assertFalse(s.server_side)
+            except (ConnectionResetError, ConnectionAbortedError) as e:
+                # sometimes windows throws ConnectionResetError during the handshake
+                sys.stdout.write(repr(e))
```

Comment:
Writing to standard output will cause the test to fail on buildbots (and then pass on rerun where stdout is expected).

If these errors are "okay", maybe we can handle them in `test_wrap_socket`?

Reply:
There's also a `support.verbose` flag that you can use to check whether additional output is expected.

Reply:
This was caused by an error in openssl.  Removed try/except.
Submitted patches for openssl
https://github.com/openssl/openssl/pull/8590
https://github.com/python/cpython-source-deps/pull/13